src/main/java/io/github/thebusybiscuit/sensibletoolbox/api/items/BaseSTBMachine.java

```diff
@@ -584,6 +584,13 @@ public void onBlockRegistered(Location location, boolean isPlacing) {
 
     @Override
     public void onBlockUnregistered(Location loc) {
+
+        // Machines broken while viewed allow items to be taken out.
+        // We need to close the inventories if opened.
+        for (HumanEntity h : getGUI().getViewers()) {
+            h.closeInventory();
+        }
```

Comment:
I think this might be re-used somewhere else, a nifty `getGUI().closeAllViews()` method would be handy.